hypha/public/utils/models.py

```diff
@@ -320,6 +320,12 @@ class Meta:
         ),
     )
 
+    header_content = models.TextField(
+        "Header content",
+        help_text=_("This will overwrite the default header, html tags is allowed."),
```

Comment:
This is good, let the default stay until a custom nav is added.

When we finally scrap the public side of Hypha the default will likely by "My dashboard etc.", as it is on all the apply pages.